motion_predict/include/motion_predict/motion_predict.hpp

```diff
@@ -54,8 +54,9 @@ namespace cv{
     \param  ax acceleration noise along x-axis (m^2/s^4)
     \param  ay acceleration noise along y-axis (m^2/s^4)
     \param  process_noise_max is the maximum process noise of the system
+    \note twist.linear vector is expected to be in the map frame and used as the yaw for future prediction.
+        pose.orientation is not meaningfully used at the moment except passing it on.
```

Comment:
This might be confusing since there is no twist or pose args for this function. Could we add a note to the `obj` comment, i.e. "\param  obj external object; expected in map frame" or similar?

Reply:
I added the comment to the obj section